docs/data/material/components/snackbars/SnackbarMaterialYouPlayground.js

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
```

Comment:
@Best-Sardar I think we shouldn't include this playground in this PR yet - at this stage of the component there aren't any visible design changes to show yet